arches/__init__.py

```diff
@@ -6,7 +6,7 @@
 except ModuleNotFoundError as e:
     print(e)
 
-VERSION = (6, 0, 0, "beta", 0)  # VERSION[3] options = "alpha", "beta", "rc", or "final"
+VERSION = (6, 0, 0, "final", 0)  # VERSION[3] options = "alpha", "beta", "rc", or "final"
```

Comment:
Could you change the version back to "beta"?

Reply:
done